BasicRdl/ViewModels/Ribbons/CategoryRibbonViewModel.cs

```diff
@@ -68,7 +68,7 @@ public static IPanelViewModel InstantiatePanelViewModel(ISession session, IThing
             var stopWatch = Stopwatch.StartNew();
             var viewModel = new CategoryBrowserViewModel(session, session.RetrieveSiteDirectory(), thingDialogNavigationService, panelNavigationService, dialogNavigationService, pluginSettingsService);
             stopWatch.Stop();
-            Logger.Info("The CategoryBrowserViewModel opened in {0} [ms]", stopWatch.Elapsed);
+            Logger.Info("The CategoryBrowserViewModel opened in {0}", stopWatch.Elapsed.ToString("g"));
```

Comment:
The [ms] part was removed, I'm not sure it was intentional.

Reply:
Are you also sure you don't want to show the milliseconds part anymore?
```
new TimeSpan(16:14:30)

g                16:14:30
G      0:16:14:30.0000000
```